README.md

```diff
@@ -10,9 +10,9 @@ custom elements.
 polydev must be built before running. The build step externalizes inline scripts
 for CSP compliance, and copies some dependencies into convenient locations.
 
-To build, run `gulp`:
+To build, run `npm run build`:
```

Comment:
What's the benefit of this over `gulp`? I haven't quite grokked yet why a package manager should be use to build the project.